exo/main.py

```diff
@@ -51,6 +51,7 @@
 parser.add_argument("--prompt", type=str, help="Prompt for the model when using --run-model", default="Who are you?")
 parser.add_argument("--tailscale-api-key", type=str, default=None, help="Tailscale API key")
 parser.add_argument("--tailnet-name", type=str, default=None, help="Tailnet name")
+parser.add_argument("--preload-models", type=str, help="Comma-separated list of models to preload")
```

Comment:
this doesn't seem to do anything

Reply:
fyi I think it's a good idea so it should be implemented

Reply:
my bad I forgot to push the latest version where I implemented it lol